src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ComplexTypeModelBinderProvider.cs

```diff
@@ -20,9 +19,7 @@ public IModelBinder GetBinder(ModelBinderProviderContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            if (context.Metadata.IsComplexType &&
-                !context.Metadata.IsCollectionType &&
-                HasDefaultConstructor(context.Metadata.ModelType.GetTypeInfo()))
+            if (context.Metadata.IsComplexType && !context.Metadata.IsCollectionType)
```

Comment:
Suggest this provider should continue to ignore all value types but I'm not 💯% convinced either way. On one hand, code in the binder can't create an instance of any value type and most value types lack settable properties. On the other, the `Model` already exists unless `Metadata.IsNullableValueType` and it _might_ have settable properties.

@rynowak @Eilon thoughts?